functions.php

```diff
@@ -48,3 +48,20 @@ function setupshop_widgets_init() {
   	'after_title'   => '</h2>', ) );
 }
 ?>
+
+<?php 
+  add_post_meta( 5 , 'setupshop_interventions_focus' , 'education' , false ); 
+  add_post_meta( 5 , 'setupshop_interventions_focus' , 'health' , false );
+  add_post_meta( 5 , 'setupshop_interventions_focus' , 'games' , false );
```

Comment:
Meaning; 
1. for the post_id, instead of the wp-id (i.e "5"), should call by action tag, the_ID() ...or are all the parameters supposed to be generic and substituted with names only where and when needed?